<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detector de Vehiculos Aerio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 241, 255);</string>
  </property>
  <widget class="QWidget" name="mainWindow">
   <widget class="QGroupBox" name="addline">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Lineas de deteccion</string>
    </property>
    <widget class="QToolButton" name="addVLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>26</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/minus-sign-of-a-line-in-horizontal-position.png</normaloff>img/minus-sign-of-a-line-in-horizontal-position.png</iconset>
     </property>
    </widget>
    <widget class="QToolButton" name="addHLine">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>26</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/horizontal.png</normaloff>img/horizontal.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Visualizar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">innerGBox-&gt;setStyleSheet(&quot;background-color: red&quot;);</string>
    </property>
    <property name="title">
     <string>Visualizar objetos</string>
    </property>
    <widget class="QCheckBox" name="flagBoxes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bounding Boxes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="flagLine">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Lineas deteccion</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableCarDetect">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>391</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableVline">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableHline">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lineas Verticales </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lineas Horizontales</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Carcount">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Contador Vehiculos</string>
    </property>
    <widget class="QLCDNumber" name="CarCount">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(115, 131, 192);</string>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="videoPanel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>681</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="start_video">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/video-player.png</normaloff>img/video-player.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="addmanualdetection">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Detecci√≥n</string>
    </property>
    <widget class="QToolButton" name="add_ManualDetection">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>26</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/selection.png</normaloff>img/selection.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="extractor_but">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>410</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/data-export.png</normaloff>img/data-export.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>29</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
